### DISCUSSION TITLE
Suggest : add database to Github Pages

### DISCUSSION THREAD

**Discussion by Rix565** [community/community#23800] [Other Feature Feedback, Questions, & Ideas]
<p>Please add databases to Github Pages.<br>
Thanks you.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Hello <a class="mention" href="/rix565">@Rix565</a>, thanks for joining the community.</p>
<p>GitHub Pages is designed to provide simple, free, static website hosting for things like documentation about projects or a site to showcase yourself or your organization. If you’re interested in a web host that provides support for databases and other options for hosting dynamically-generated content, there are many, many options to choose from including services like <a href="https://www.heroku.com/">Heroku</a> and <a href="https://glitch.com/">Glitch</a>.</p>
<p>I hope that helps!</p>

**Comment by thiagorussi** [comment]
seria uma boa
